### DISCUSSION TITLE
Save servings in Meal Plan

### DISCUSSION THREAD

**Discussion by FuzzyWereBeast** [mealie-recipes/mealie#4698] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

When I add recipes to the meal plan, the serving amounts are not saved.  Right now the system is set up where you can hit the 3 dot menu and select "add to list".  Unfortunately, this is not usable unless your recipe is saved with the appropriate amount of servings.  A lot of my recipes are originally marked to serve 4.  However, I normally want to make 6 for some extras.  Because of this, I need to open each recipe individually to make this adjustment before adding to the list.

### Please provide a concise description of the feature that would resolve your issue.

Save the amount of servings to the Meal Plan.

### Please consider and list out some caveats or tradeoffs made in your design decision

Solution #1
Have the amount of servings or scale of the whole recipe saved in the meal plan.

Solution #2
Instead of having the scale on the shopping list show by whole recipe, adjust this to servings.

Solution #3
Allow manual editing of the scale on the shopping list so you can do partials.  Right now you can only scale by a factor of 1.  I'd like to be able to scale by 1.5 or just enter 1.5 for 6 servings from a 4 serving recipe.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by FuzzyWereBeast** [comment]
On the same note, when you access the recipe from the Meal Plan, the scale should automatically be set to what was saved in the Meal Plan.

**Comment by franckleveque** [comment]
We also have a similar problem where our recipe are made for one portion but our family is 5 people. So we need each time to adapt the number of serving to 5. It would be great to have a preferred portion size in the family or user settings.
By the way, I don't understand your solution 3 as it already seems to work the way you expect. it only set the initial portion to the one in the recipe. Afterwards, you can decrement / increment the recipe by one portions even if some ingredient can't be divided (ie: in the case of a pithiviers where it need 2 puff patries to be made with 6 portion, when I increment (7) or decrement (5) by one portion I have partial puff patries (1 2/3 or 2 1/3) used which is incoherent according to me.)

**Comment by FuzzyWereBeast** [comment]
Clarification for number 3.

When you add a 4 person recipe to the shopping list, you will see at the bottom of the list the recipe name followed by "- 1 +". Hitting the plus will take change the number to a 2, meaning 8 servings. Unless I missed something, you can only advance by whole numbers causing the servings to be a multiple of 4 in this example.

**Reply by franckleveque** [reply]
Hi, Thanks for your answer, It seems we were not talking about the same place. I was talking about the recipe servings button. I just checked the shopping list, and i'm surprised that it not the portion size displayed but the number of times you want the recipe added.

**Comment by mdecker10** [comment]
I think this would improve the workflow for meal planning and shopping. Currently, we do weekly meal planning and we try to adjust the scale of recipes to fit the number of people eating.  We also like leftovers, so we are constantly scaling a recipe up or down.  I've been testing the individual ingredient amounts feature and can see how you can scale up or down incrementally when looking at a recipe.  I also see how the scaled recipe ingredients could be added to a shopping list, which is great.  

Thinking about two different workflows:
1) I have to bring a recipe to a holiday dinner for 7.  In Mealie, I find a recipe that serves 4, I can scale it to serve 7, and finally add it to the shopping list.  This seems to work very well.
2) I am planning a holiday party for New Years for 7 people.  I find great recipes in Mealie, scale them to feed 7, and add to a meal plan.  As you mentioned, the scaled amount aren't stored in the Meal Plan.  This causes my shopping list to be incorrect.

TBH, until this thread, I had never seen the ability to scale an entire recipe up or down from the shopping list.  That's cool if I want to simply double/triple the entire recipe.  But, if I need to add more servings to the list, more granular control would be awesome.
